carddav2fb.php

```diff
@@ -504,7 +508,8 @@ public function upload_to_fb() {
     }
     catch (Exception $e)
     {
-      print "  ERROR: " . $e->getMessage() . PHP_EOL;     // show the error message in anything failed
+//      print "  ERROR: " . $e->getMessage() . PHP_EOL;     // show the error message in anything failed
+      print "  ERROR: " . $e . PHP_EOL;     // show the error message in anything failed
```

Comment:
> Am 18.12.2015 um 14:59 schrieb Jens Maus <notifications@github.com>:
> 
> In carddav2fb.php:
> 
> > @@ -504,7 +508,8 @@ public function upload_to_fb() {
> >      }
> >      catch (Exception $e)
> >      {
> > -      print "  ERROR: " . $e->getMessage() . PHP_EOL;     // show the error message in anything failed
> > +//      print "  ERROR: " . $e->getMessage() . PHP_EOL;     // show the error message in anything failed
> > +      print "  ERROR: " . $e . PHP_EOL;     // show the error message in anything failed
> 
> can you please explain why you replace $e->getMessage() here with plainly $e? I think it still should use $e->getMessage().

Ah sorry - debug artefact. I needed to know where the error came from. $e-getMessage() is better here.